queue/python/queue.py

```diff
@@ -0,0 +1,32 @@
+class Queue:
+    def __init__(self):
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.pyjava`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpvd7j5r87/queue/python/queue.py
+++ b/tmp/tmpvd7j5r87/queue/python/queue.py
@@ -15,7 +15,7 @@
         return len(self.items)
     
     def front(self):
-        return self.items[self.size()-1] 
+        return self.items[self.size()-1]
 
 def main():
     q=Queue()
```